### ISSUE TITLE
My.Youfone.nl blocked

### ISSUE THREAD

**Issue by Pammetje** [geertmeersman/youfone#115]
My.Youfone.nl is blocked , I have 3 accounts
Can there maybe be an option added that the accounts are queried only once or twice a day ? so that the sites is not blocked ?

**Comment by geertmeersman** [comment]
Hi, sure I will try to put sth in place for this, so you can specify the update interval yourself.

**Comment by Pammetje** [comment]
Thank you very much for your work, I will test it as soon as I am unblocked

**Comment by geertmeersman** [comment]
Make sure to disable your 3 accounts in HA in the meantime, this can speed up the unblocking

**Comment by Pammetje** [comment]
Hi I do not see the configuration options, is this because the configuration are disabled at this moment, or do I see the configuration options only If I create a new hub/configuration ? I do not see the option when I go to settings -> Devices & Services ->  integration  YouFone -> hubs -> configuration

**Comment by geertmeersman** [comment]
You only see them when adding or when it is enabled

**Comment by Pammetje** [comment]
The integration is not working, I did not see any updates, when I delete and create a new/account, I get "unexpected error"

**Comment by geertmeersman** [comment]
Can you paste the error from the log

**Comment by geertmeersman** [comment]
This is how it looks like at my side. I started with a fresh HA installation to check to be sure

![image](https://github.com/geertmeersman/youfone/assets/1844091/6dd0b2a3-3cb5-4d92-a7c4-5230c9b2a3d4)


You can always join on discord, might ease things: https://discord.com/invite/JpjHptEN2D

**Comment by Pammetje** [comment]
It looks the same on my system. I removed the configurations/HUBs. Uninstalled the Youfone integration in HACS. Reboot the system. (Checked there was no Youfone folder in custom_components. Reinstalled the integration from HACS. Added a configuration. Still the same issue. Did you also check a NL Youfone serivce ? 



![image](https://github.com/geertmeersman/youfone/assets/16224006/6423e4cb-8639-47a5-91b0-c2f5e77024fe)

**Comment by Pammetje** [comment]
where can I find the debug logs ?

**Comment by Pammetje** [comment]
This one https://github.com/myTselection/youfone_be is also not working, so maybe something has changed at the NL Youfone service

**Comment by geertmeersman** [comment]
You can put this in your configuration
Then add the account and in your home-assistant.log file you will find the related logs
```
logger:
  default: warning
  logs:
    custom_components.youfone: debug
```

**Comment by geertmeersman** [comment]
I just checked my.youfone.nl and indeed it is using a new API.
So I can help out by writing new calls into the integration, but then I would need a login.
Or you can come to discord and we work it out together.
The Youfone BE endpoint will probably be upgraded once in NL all is confirmed working fine, which I will then be able to test with my account

**Comment by Pammetje** [comment]
2024-02-16 15:30:24.235 DEBUG (SyncWorker_35) [custom_components.youfone.client] [YoufoneClient|login|authenticate] Calling POST https://my.youfone.nl/prov/MyYoufone/MyYOufone.Wcf/v2.0/Service.svc/json/login with {'request': {'Login': 'xxxxxxxxxxx@gmail.com', 'Password': '***FILTERED***'}}
2024-02-16 15:30:24.334 DEBUG (SyncWorker_35) [custom_components.youfone.client] [YoufoneClient|login|authenticate] http status code = 200 (expecting None)
2024-02-16 15:30:24.334 DEBUG (SyncWorker_35) [custom_components.youfone.client] [YoufoneClient|login|authenticate] Response:
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <base href="/">
    <meta charset="utf-8" />
<meta name="Generator" content="Drupal 10 (https://www.drupal.org)" />
<meta name="MobileOptimized" content="width" />
<meta name="HandheldFriendly" content="true" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<style>
@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCRc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fABc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0301, U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCBc4AMP6lbBP.woff2) format('woff2');unicode-range:U+1F00-1FFF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fBxc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0370-03FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCxc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01A1, U+01AF-01B0, U+0300-0301, U+0303-0304, U+0308-0309, U+0323, U+0329, U+1EA0-1EF9, U+20AB;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fChc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0100-02AF, U+0304, U+0308, U+0329, Um+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fBBc4AMP6lQ.woff2) format('woff2');unicode-range:U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu72xKKTU1Kvnz.woff2) format('woff2');unicode-range:U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu5mxKKTU1Kvnz.woff2) format('woff2');unicode-range:U+0301, U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7mxKKTU1Kvnz.woff2) format('woff2');unicode-range:U+1F00-1FFF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu4WxKKTU1Kvnz.woff2) format('woff2');unicode-range:U+0370-03FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7WxKKTU1Kvnz.woff2) format('woff2');unicode-range:U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01A1, U+01AF-01B0, U+0300-0301, U+0303-0304, U+0308-0309, U+0323, U+0329, U+1EA0-1EF9, U+20AB;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7GxKKTU1Kvnz.woff2) format('woff2');unicode-range:U+0100-02AF, U+0304, U+0308, U+0329, U+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2) format('woff2');unicode-range:U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fCRc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fABc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0301, U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fCBc4AMP6lbBP.woff2) format('woff2');unicode-range:U+1F00-1FFF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fBxc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0370-03FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fCxc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169, U+01A0-01A1, U+01AF-01B0, U+0300-0301, U+0303-0304, U+0308-0309, U+0323, U+0329, U+1EA0-1EF9, U+20AB;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fChc4AMP6lbBP.woff2) format('woff2');unicode-range:U+0100-02AF, U+0304, U+0308, U+0329, U+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;}@font-face{font-family:'Roboto';font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmEU9fBBc4AMP6lQ.woff2) format('woff2');unicode-range:U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;}@font-face{font-family:'Material Icons';font-style:normal;font-weight:400;src:url(https://fonts.gstatic.com/s/materialicons/v140/flUhRq6tzZclQEJ-Vdg-IuiaDsNcIhQ8tQ.woff2) format('woff2');}.material-icons{font-family:'Material Icons';font-weight:normal;font-style:normal;font-size:24px;line-height:1;letter-spacing:normal;text-transform:none;display:inline-block;white-space:nowrap;word-wrap:normal;direction:ltr;-webkit-font-feature-settings:'liga';-webkit-font-smoothing:antialiased;}.mat-typography{font:400 14px/20px Roboto,sans-serif;letter-spacing:.0178571429em}@font-face{font-family:Outfit;font-style:normal;font-weight:100;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:200;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:300;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:400;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:500;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:600;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:700;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:800;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}@font-face{font-family:Outfit;font-style:normal;font-weight:900;font-display:swap;src:url(https://fonts.gstatic.com/s/outfit/v2/QGYvz_MVcBeNP4NJtEtq.woff2) format('woff2');unicode-range:U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD}*{box-sizing:border-box;font-family:Outfit,sans-serif}html,body{margin:0;padding:0;width:100%;height:100vh;position:relative;font-family:Outfit,sans-serif;font-size:15px;font-weight:400;font-stretch:normal;font-style:normal;line-height:1.73;letter-spacing:.1px}body{min-height:100vh;min-height:-webkit-fill-available;overflow-x:hidden;background-color:#e5e5e5!important}html{height:-webkit-fill-available}.mat-typography{font:400 18px/22px Outfit,sans-serif;letter-spacing:normal} app-root{height:100%}
.dialog-off-canvas-main-canvas{height:100%!important}
#freshwidget-button{display: none !important; }
  </style>
<link rel="icon" href="/core/misc/favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Page not found | youfone</title>
    <link rel="stylesheet" media="all" href="/sites/default/files/css/css_J2He_dRQ4mMxU8hASrSzfeeD09PuI9PNLGhcj1Rzmp8.css?delta=0&amp;language=en&amp;theme=youfone&amp;include=eJwrriwuSc3VT0osTtWpzC9Ny89L1c_JT8_M06lMi08syNSHUPGJKbmZebol-fk5SYlFcJW5lfFQJlBRAQB7Oh4G" />
<link rel="stylesheet" media="all" href="/sites/default/files/css/css_6V89LhEkSb0jP8Lpe1_BKXQ_-2f2gTKk_caqtZN9fiU.css?delta=1&amp;language=en&amp;theme=youfone&amp;include=eJwrriwuSc3VT0osTtWpzC9Ny89L1c_JT8_M06lMi08syNSHUPGJKbmZebol-fk5SYlFcJW5lfFQJlBRAQB7Oh4G" />

    
    <link rel="icon" type="image/x-icon" href="/themes/youfone/images/favicon/favicon.ico">
    <link rel="apple-touch-icon" sizes="180x180" href="/themes/youfone/images/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/themes/youfone/images/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/themes/youfone/images/favicon/favicon-16x16.png">
    <link rel="manifest" href="/themes/youfone/images/favicon/site.webmanifest">
    
<script data-ot-ignore>
  let googleTagManagerID = function () {
    if ( location.href.indexOf('youfone') > -1 ) {
      return 'GTM-TCQ2ZVC';
    } else if ( location.href.indexOf('youstaging') > -1 ) {
      return 'GTM-KT7QLZM';
    }
    return 'GTM-K8N8X9T';
  }();
  (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.setAttribute('class','optanon-category-C0001');
        j.async = true;j.src=
      'https://tag.youfone.nl/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
  })(window,document,'script','dataLayer', googleTagManagerID);
</script>



<noscript>
  <iframe id="gtm-iframe" height="0" width="0" style="display:none;visibility:hidden"></iframe>
</noscript>
<script>
  var iframe = document.getElementById("gtm-iframe");
  if(iframe){
    iframe.src = "https://www.googletagmanager.com/ns.html?id=" + googleTagManagerID;
  }
</script>



<script> var onetrust_id = (( location.href.indexOf('youfone') > -1 ) ? '3b35fed9-6a85-4de9-83ab-efda375c8397' : ( location.href.indexOf('youstaging') > -1 ) ? '4f5a0c8b-a835-43ea-b941-1a9a131f0e44' : 'e7e97271-8dc4-4810-a9c5-89a787f5e7a1'); var script2 = document.createElement('script'); script2.src = 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js'; script2.type = 'text/javascript'; script2.setAttribute('data-domain-script', onetrust_id); script2.id = 'onetrust'; document.head.appendChild(script2); var script3 = document.createElement('script'); script3.textContent = 'function OptanonWrapper() { }'; document.head.appendChild(script3); </script>


<style>.xnpe_async_hide{opacity:0 !important}</style>
<script class="optanon-category-C0001">
    !function(e,n,t,i,r,o){function s(e){if("number"!=typeof e)return e;var n=new Date;return new Date(n.getTime()+1e3*e)}var a=4e3,c="xnpe_async_hide";function p(e){return e.reduce((function(e,n){return e[n]=function(){e._.push([n.toString(),arguments])},e}),{_:[]})}function m(e,n,t){var i=t.createElement(n);i.src=e;var r=t.getElementsByTagName(n)[0];return r.parentNode.insertBefore(i,r),i}function u(e){return"[object Date]"===Object.prototype.toString.call(e)}o.target=o.target||"https://api.exponea.com",o.file_path=o.file_path||o.target+"/js/exponea.min.js",r[n]=p(["anonymize","initialize","identify","getSegments","update","track","trackLink","trackEnhancedEcommerce","getHtml","showHtml","showBanner","showWebLayer","ping","getAbTest","loadDependency","getRecommendation","reloadWebLayers","_preInitialize","_initializeConfig"]),r[n].notifications=p(["isAvailable","isSubscribed","subscribe","unsubscribe"]),r[n].segments=p(["subscribe"]),r[n]["snippetVersion"]="v2.7.0",function(e,n,t){e[n]["_"+t]={},e[n]["_"+t].nowFn=Date.now,e[n]["_"+t].snippetStartTime=e[n]["_"+t].nowFn()}(r,n,"performance"),function(e,n,t,i,r,o){e[r]={sdk:e[i],sdkObjectName:i,skipExperiments:!!t.new_experiments,sign:t.token+"/"+(o.exec(n.cookie)||["","new"])[1],path:t.target}}(r,e,o,n,i,RegExp("__exponea_etc__"+"=([\\w-]+)")),function(e,n,t){m(e.file_path,n,t)}(o,t,e),function(e,n,t,i,r,o,p){if(e.new_experiments){!0===e.new_experiments&&(e.new_experiments={});var l,f=e.new_experiments.hide_class||c,_=e.new_experiments.timeout||a,g=encodeURIComponent(o.location.href.split("#")[0]);e.cookies&&e.cookies.expires&&("number"==typeof e.cookies.expires||u(e.cookies.expires)?l=s(e.cookies.expires):e.cookies.expires.tracking&&("number"==typeof e.cookies.expires.tracking||u(e.cookies.expires.tracking))&&(l=s(e.cookies.expires.tracking))),l&&l<new Date&&(l=void 0);var d=e.target+"/webxp/"+n+"/"+o[t].sign+"/modifications.min.js?http-referer="+g+"&timeout="+_+"ms"+(l?"&cookie-expires="+Math.floor(l.getTime()/1e3):"");"sync"===e.new_experiments.mode&&o.localStorage.getItem("__exponea__sync_modifications__")?function(e,n,t,i,r){t[r][n]="<"+n+' src="'+e+'"></'+n+">",i.writeln(t[r][n]),i.writeln("<"+n+">!"+r+".init && document.writeln("+r+"."+n+'.replace("/'+n+'/", "/'+n+'-async/").replace("><", " async><"))</'+n+">")}(d,n,o,p,t):function(e,n,t,i,r,o,s,a){o.documentElement.classList.add(e);var c=m(t,i,o);function p(){r[a].init||m(t.replace("/"+i+"/","/"+i+"-async/"),i,o)}function u(){o.documentElement.classList.remove(e)}c.onload=p,c.onerror=p,r.setTimeout(u,n),r[s]._revealPage=u}(f,_,d,n,o,p,r,t)}}(o,t,i,0,n,r,e),function(e,n,t){var i;e[n]._initializeConfig(t),(null===(i=t.experimental)||void 0===i?void 0:i.non_personalized_weblayers)&&e[n]._preInitialize(t),e[n].start=function(i){i&&Object.keys(i).forEach((function(e){return t[e]=i[e]})),e[n].initialize(t)}}(r,n,o)}(document,"exponea","script","webxpClient",window,{
    target: "https://analytics-api.youfone.nl",
    token: "d3e5387e-a1ae-11ed-8703-ae7152078daf",
     experimental: {
        non_personalized_weblayers: true
    },
    new_experiments: { mode: "sync", timeout: 1000},
    // replace with current customer ID or leave commented out for an anonymous customer
    // customer: window.currentUserId,
    track: {
        visits: true,
    },
});
</script>

<script type="text/plain" class="optanon-category-C0003">
exponea.start();
</script>

<link rel="preconnect" href="https://chat.kindlycdn.com" />  </head>
  <body class="mat-typography">
        <a href="#main-content" class="visually-hidden focusable">
      Skip to main content
    </a>
    
      <div class="dialog-off-canvas-main-canvas" data-off-canvas-main-canvas>
              <app-root></app-root>
    
  </div>

    
    <script src="/themes/youfone/my-app/build/runtime.d802ec5b641e6b5a.js?v=1.x" type="module"></script>
<script src="/themes/youfone/my-app/build/polyfills.64db045ecf3434ae.js?v=1.x" type="module"></script>
<script src="/themes/youfone/my-app/build/main.c883a12bd6ee2c1c.js?v=1.x" type="module"></script>

  </body>
</html>

2024-02-16 15:30:24.341 DEBUG (MainThread) [custom_components.youfone.config_flow] Expecting value: line 1 column 1 (char 0)

**Comment by Pammetje** [comment]
Ok I think I will just wait till the .be youfone api is also updated, I hope the logs will help you

**Comment by geertmeersman** [comment]
Yes, it's clear that the old API endpoint is not operational anymore. So I either need an NL login, or just patience :-)

**Comment by DrBlokmeister** [comment]
For me the Youfone integration stopped working as well. From what I read above, I understand that this integration has stopped working for everyone due to a new API. Is this correct, or should a separate issue be opened for this?

**Comment by geertmeersman** [comment]
No new issue is needed.

**Comment by DrBlokmeister** [comment]
Thanks for the response! Do you think this will be an easy fix, or will it take some time to implement? Not looking for exact answers, more along the lines of days/weeks/months (I understand that 'geen rechten kunnen worden ontleend aan de uitspraak' 😄).

**Comment by geertmeersman** [comment]
Indien iemand me tijdelijk zijn account (met aangepast wachtwoord) zou kunnen bezorgen dan kan ik dit waarschijnlijk snel fixen 😀😀
Maar zolang ik niet op het systeem kan aanmelden dan kan ik ook niets ontwikkelen 🤪

**Comment by geertmeersman** [comment]
Status update:  Login seems operational, if all goes well a fully working version will be available by end of this week

**Comment by Pammetje** [comment]
Thanks for all the work, sorry I could not provide an NL Youfone account. My personal account is at ODIDO not at Youfone (I am using your integration for my family members with Youfone accounts) If I can help with testing let me know

**Comment by geertmeersman** [comment]
Hi, sure you can test, possible to join on discord? Then I can tell you how to test

**Comment by geertmeersman** [comment]
https://github.com/geertmeersman/youfone/releases/tag/v0.8.0-beta.1

Have not been able to test it but it should already give something 😀

**Comment by geertmeersman** [comment]
It will show up in hacs if you check the beta switch
